include/model/Note.hpp

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#include <chrono>
+#include <iostream>
+#include <vector>
+
+namespace doma {
+namespace model {
+
+using ModelClock = std::chrono::system_clock;
+using Timestamp = ModelClock::time_point;
```

Comment:
please create `Timestamp.h` with `using Timestamp = ...` and include the header here

Reply:
done